cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -78,126 +80,144 @@ class CorrectIlluminationCalculate(cpm.Module):
+   often applied by subtraction using the **CorrectIlluminationApply**
+   module. *Note:* if you will be using the *{DOS_SUBTRACT}* option in
+   the **CorrectIlluminationApply** module, you almost certainly do not
+   want to rescale the illumination function.
+
+Please note that if a mask was applied to the input image, the pixels
+outside of the mask will be excluded from consideration. This is useful,
+for instance, in cases where you have masked out the well edge in an
+image from a multi-well plate; the dark well edge would distort the
+illumination correction function along the interior well edge. Masking
+the image beforehand solves this problem.
+""".format(**{
+                "IC_REGULAR": IC_REGULAR,
+                "EA_ALL": EA_ALL,
+                "EA_EACH": EA_EACH,
+                "SM_NONE": SM_NONE,
+                "IC_BACKGROUND": IC_BACKGROUND,
+                "DOS_SUBTRACT": DOS_SUBTRACT
+            })
+        )
 
         self.dilate_objects = cps.Binary(
-                "Dilate objects in the final averaged image?", False, doc='''
-            <i>(Used only if the Regular method is selected)</i><br>
-            For some applications, the incoming images are binary and each object
-            should be dilated with a Gaussian filter in the final averaged
-            (projection) image. This is for a sophisticated method of illumination
-            correction where model objects are produced.
-            Select <i>%(YES)s</i> to dilate objects for this approach.
-            ''' % globals())
+            "Dilate objects in the final averaged image?", False, doc="""\
+*(Used only if the Regular method is selected)*
+For some applications, the incoming images are binary and each object
+should be dilated with a Gaussian filter in the final averaged
+(projection) image. This is for a sophisticated method of illumination
+correction where model objects are produced. Select *%(YES)s* to dilate
+objects for this approach.
+""" % globals())
 
         self.object_dilation_radius = cps.Integer(
-                "Dilation radius", 1, 0, doc='''
-            <i>(Used only if the "%(IC_REGULAR)s" method and dilation is selected)</i><br>
-            This value should be roughly equal to the original radius of the objects''' % globals())
+            "Dilation radius", 1, 0, doc="""\
+*(Used only if the “%(IC_REGULAR)s” method and dilation is selected)*
+This value should be roughly equal to the original radius of the objects
+""" % globals())
 
         self.block_size = cps.Integer(
-                "Block size", 60, 1, doc='''
-            <i>(Used only if "%(IC_BACKGROUND)s" is selected)</i><br>
-            The block size should be large enough that every square block of pixels is likely
-            to contain some background pixels, where no objects are located.''' % globals())
+            "Block size", 60, 1, doc="""\
+*(Used only if “%(IC_BACKGROUND)s” is selected)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.